Added Bulgarian Translation

twlvnn opened the pull request.

twlvnn committed:
Added Bulgarian Translation

twlvnn committed:
Updated Bulgarian Translation

zefr0x commented on po/bg.po:
```diff
@@ -0,0 +1,121 @@
+
+#: data/io.github.zefr0x.hashes.desktop.in:13
+msgid "hash;hashcat;john;detect;identify"
+msgstr "hash;hashcat;john;detect;identify;хеш;засичане;идентифициране;"
```
This should not include English key words, just Bulgarian as possible: https://github.com/zefr0x/hashes/pull/6#discussion_r1548860728

zefr0x commented on po/bg.po:
```diff
@@ -0,0 +1,121 @@
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:12
+msgid "📺 Popularity Ratings - Most popular hashes showed first."
+msgstr ""
+"Класификация на популярността - Най-популярните хешове се показват първи."
```
looks like you missed the emoji here

twlvnn committed:
Corrected translation mistakes

twlvnn commented:
Thank you for the feedback zefr0x, I've corrected the mistakes and fixed another mistake I made with the hyphenation. Let me know if there's anything I missed!

zefr0x commented:
Thanks to you, Bulgarians can use the application in their native language
